contrib/system-setup.bat

```diff
@@ -113,8 +113,8 @@ greextra.ttf ^
 gregorio.ttf ^
 gregorio-op.ttf ^
 gresym.ttf ^
-parmesan.ttf ^
-parmesan-op.ttf ^
+granapadano.ttf ^
+granapadano-op.ttf ^
 gregall.ttf ^

```

Comment:
I would leave the parmesan names here and in the system-setup.command below.  That way when it comes to diagnosing problems, we can see if the old file names are returning results (indicating possible version conflicts).  That's why `gregoriotex-ictus.tex` is still in the list.